docs/s2i.md

```diff
@@ -0,0 +1,74 @@
+# s2i
+
+Containers Tekton Tasks
+
+---
+
+# `s2i` Tekton Task
```

Comment:
Let me suggest a different tile: "# `s2i` Builder Images"

Here we explain each language ecosystem comes with a predefined set of builder images, supported by Red Hat.